pkg/proxy/volume.go

```diff
@@ -308,3 +314,18 @@ func (ops V2DataEngineProxyOps) VolumeSnapshotMaxSizeSet(ctx context.Context, re
 	/* TODO: Implement this */
 	return &emptypb.Empty{}, nil
 }
+
+func (p *Proxy) RemountReadOnlyVolume(ctx context.Context, req *rpc.RemountVolumeRequest) (resp *emptypb.Empty, err error) {
+	volumeName := req.VolumeName
+	volumeNameSHA := sha256.Sum256([]byte(volumeName))
+	volumeNameSHAStr := hex.EncodeToString(volumeNameSHA[:])
+
+	volumeGlobalMountPath := fmt.Sprintf(types.GlobalMountPathFormat, volumeNameSHAStr)
+
+	cmd := exec.CommandContext(ctx, "mount", "-o", "remount,rw", volumeGlobalMountPath)
```

Comment:
Do we need to use `NamespaceExecutor` in `"github.com/longhorn/go-common-libs/ns"` instead?

Reply:
It is manipulating the host directory, I think there is no need to use `NamespaceExecutor`
Is it correct?

Reply:
This command is executed in the host namespace rather than the current container namespace. I think it should use `NamespaceExecutor`. This is similar to `iscsiadm` cmds.

Reply:
Fixed, thanks!